framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesError.java

```diff
@@ -27,7 +27,7 @@ public class DependentTypesError {
 
     /** Regular expression for unparsing string representations of this class (gross). */
     private static final Pattern ERROR_PATTERN =
-            Pattern.compile("\\[error for expression: (.*); error: (.*)\\]");
+            Pattern.compile("\\[error for expression: (.*); error: ([\\s\\S]*)\\]");
```

Comment:
I wonder whether the expression could also span multiple lines? Please add a comment either way, to make it easier to read.